### PR TITLE
Added new mappings

### PR DESCRIPTION
**Pull request by imm007**
Added new mappings for:
- findings (alerts, offense etc)
- MITRE tactics/techniques
- flow/event direction
- devicetype
- log source type name
- severity/risk

### CODE REVIEW

**Review by pramodgslab — APPROVED** [review]

**Review by pramodgslab — APPROVED** [review]

File: stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json [hunk 1]
```diff
@@ -138,5 +141,27 @@
...
+    "x-ibm-finding": {
+        "cim_type": "endpoint",
+        "fields": {
+          "ss_name": ["ss_name"],
+          "severity": ["severity"]
+        }
     }
-}
\ No newline at end of file
+}
```
Comment (mdazam1942) [inline stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json:167]: I think you are not supposed to make changes in `cim_from_stix_map.json`. we need to have a `from_stix_map.json` file inside `stix_shifter_modules/splunk/stix_translation/json`. Make a copy of `cim_from_stix_map.json` and put it inside splunk module and make changes there.

File: stix_shifter_utils/modules/cim/stix_translation/json/cim_select_fields.json [hunk 2]
```diff
@@ -11,6 +11,18 @@
...
+        "EventName",
+        "ss_name",
+        "TacticId",
+        "Tactic",
+        "TechniqueId",
+        "Technique"
     ]
-}
\ No newline at end of file
+}
```
Comment (mdazam1942) [inline stix_shifter_utils/modules/cim/stix_translation/json/cim_select_fields.json:28]: same goes to this `cim_select_fields.json` file. make a copy and put that inside splunk module.

File: stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json [hunk 3]
```diff
@@ -255,5 +255,79 @@
...
+  "Direction": [
+    {
+      "key": "x-splunk.direction",
+      "object": "x-splunk"
+    }
+  ],
+  "severity": [
+    {
+      "key": "x-splunk.severity",
+      "object": "x-splunk"
```
Comment (mdazam1942) [inline stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json:286]: redundant object. you don't need to put severity in `x-splunk.severity`. you can just use `x-ibm-finding.severity`

**Review by mdazam1942 — CHANGES_REQUESTED** [review]
Build failed because unittest didn't pass. 

We don't want to use `cim_from_stix_map.json` for splunk module anymore. Please make sure splunk module has its own copy of from_stix_map.json. Let us know if you have question on the comments that I made.

File: stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json (hunk 3 above)
Comment (imm007) [reply stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json:286]: okay. working on it.

File: stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json (hunk 1 above)
Comment (pramodgslab) [reply stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json:167]: @mdazam1942 
Okay,Noted. We have resolved unittest. we'll move mapping files in splunk module and will push changes

File: stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json (hunk 3 above)
Comment (pramodgslab) [reply stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json]: @mdazam1942  Resolved redundant object severity.

File: stix_shifter_utils/modules/cim/stix_translation/json/cim_select_fields.json (hunk 2 above)
Comment (pramodgslab) [reply stix_shifter_utils/modules/cim/stix_translation/json/cim_select_fields.json:16]: Resolved

File: stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json (hunk 1 above)
Comment (pramodgslab) [reply stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json:145]: Moved mapping files in splunk module, please review

File: stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json [hunk 4]
```diff
@@ -10,7 +10,10 @@
     "domain-name": {
         "cim_type": "flow",
         "fields": {
-            "value": "url"
+            "value": [
+                "host",
+                "url"
+            ]
```
Comment (mdazam1942) [inline stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json:16]: is this mapping needed?

Comment (imm007) [reply stix_shifter_utils/modules/cim/stix_translation/json/cim_from_stix_map.json]: removed the unwanted mappings

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**